hpx/util/unwrapped.hpp

```diff
@@ -522,6 +580,7 @@ namespace hpx { namespace util
         return unwrapped(unwrapped(std::forward<Future>(f)));
     }
 
+    /// \copydoc unwrapped2
```

Comment:
I tested it and doxygen generates the documentation for the function correctly:

![screenshot_5](https://cloud.githubusercontent.com/assets/1146834/26590150/213736e8-455a-11e7-831c-238c8d7e1cf7.png)